ui/imports/shared/popups/CommunityAssetsInfoPopup.qml

```diff
@@ -1,4 +1,5 @@
-import QtQuick 2.14
+import QtQuick 2.15
+import QtQuick.Controls 2.15
```

Comment:
is it needed?

Reply:
Yes, see the commit description -> it was a syntax error due to which the popup was missing the "OK" button